php_version_resolver_test.go

```diff
@@ -0,0 +1,181 @@
+package composer_test
+
+import (
+	"io/ioutil"
```

Comment:
same as above, can we axe usage of `io/ioutil`?

Reply:
Thanks for the catch, I will do this. https://pkg.go.dev/io/ioutil